src/message/MessageFull.js

```diff
@@ -49,3 +55,10 @@ export default class MessageFull extends React.PureComponent {
     );
   }
 }
+
+export default connect(
```

Comment:
Is this connect necessary? Doesn't seem like the state or callbacks are being used -- or am I missing something?